### DISCUSSION TITLE
Is there a way to adjust wallbash colors?

### DISCUSSION THREAD

**Discussion by ljijljijljijl** [prasanthrangan/hyprdots#919] [Q&A] [answered]
Hello! After i had recently updated Hyprdots, wallbash colors became too dark and in some cases terminal text was almost unreadable. I am using custom wallpaper, but i noticed it with some other standard wallpapers especially dark ones. Is there a way to adjust wallbash colors somehow and make them a bit brighter?

Before update:
![240215_21h04m11s_screenshot](https://github.com/prasanthrangan/hyprdots/assets/63408677/bb0083ac-3862-4231-aa80-c8428a34df58)

After update:
![240220_15h08m41s_screenshot](https://github.com/prasanthrangan/hyprdots/assets/63408677/396ae21c-2401-4e7f-9009-c4b3e0e50434)

**Comment by kRHYME7** [comment, marked as answer]
@ljijljijljijl  You can try to adjust this ```  ~/.config/hypr/wallbash/kitty.dcol ``` 

I recently adjusted it in this PR https://github.com/prasanthrangan/hyprdots/pull/910


the line you're looking for is the foreground 

or you can try to use this configuration. 
```
$HOME/.config/kitty/themes/Wall-Dcol.conf|${ScrDir}/wallbashkitty.sh
foreground         #<wallbash_pry2>
background         #<wallbash_0xa1>
cursor             #<wallbash_txt1> 

active_tab_foreground     #<wallbash_pry0>
active_tab_background     #<wallbash_pry2>
inactive_tab_foreground   #<wallbash_pry2>
inactive_tab_background   #<wallbash_pry0>

active_border_color   #<wallbash_pry1>
inactive_border_color #<wallbash_pry2>
bell_border_color     #<wallbash_1xa4>

    # Commands
color2       #<wallbash_txt0>
 
    # Prompt
color7       #<wallbash_pry2>
color4       #<wallbash_2xa4>
color0       #<wallbash_txt2>

    # Color Overrides
# color1       #<wallbash_txt1>
# color3       #<wallbash_1xa3>
# color5       #<wallbash_2xa2>
# color6       #<wallbash_2xa3>
# color8       #<wallbash_1xa2>
# color9       #<wallbash_pry1>
# color10      #<wallbash_2xa4>
# color11      #<wallbash_1xa3>
# color12      #<wallbash_2xa1>
# color13      #<wallbash_2xa2>
# color14      #<wallbash_2xa3>
# color15      #<wallbash_pry2>

```

**Reply by ljijljijljijl** [reply]
Thank you very much!

**Reply by ljijljijljijl** [reply]
Ended up using Gruvbox-Retro colors for Kitty, they work great with wallpaper i'm currently using. Is there a proper way to specify what i want and don't want to theme with wallbash?

**Reply by kRHYME7** [reply]
Yes, maybe for now you can comment out lines below the very first line. 

![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/9cc3f9f7-71d4-4cda-bf4d-b6c64674160a)


Currently writing a script to toggle at least 3 terminal preset colors based on wallpaper to have at least more options for visibility. (Can be added as key bind)

**Reply by ljijljijljijl** [reply]
Thanks again for your help!

**Reply by prasanthrangan** [reply]
Hey @ljijljijljijl and @kRHYME7 ,
To disable the wallbash colors for a specific app, I would recommend renaming the file extension name (.dcol) to something else, like `~/.config/hypr/wallbash/kitty.dcol` to `~/.config/hypr/wallbash/kitty.t2`

**Reply by kRHYME7** [reply]
@prasanthrangan I'll soon add an option to disable individual wallbash related theming on hyprdots-ctl. For consistency on this repo.
